ghost/admin/app/components/tabs/tabs.hbs

```diff
@@ -0,0 +1,30 @@
+<div ...attributes>
```

Comment:
Wasn't added styles for this component as we don't have a predefined set of possible variants. It gives opportunity to style it for each usage